configs/exec/README.md

```diff
@@ -0,0 +1,12 @@
+# exec configuration readme 
+  * comment : `<today>, differentiate the same build in server ` 
+  * advance : `true/false, enable will output the debug log and auto generate the mitmdump flow fliles` 
```

Comment:
* advance : `true/false, by enabling this option, it will output the debug log and generate the mitmdump flow files automatically`